node_modules/oae-principals/emailTemplates/invite.html.jst

```diff
@@ -0,0 +1,195 @@
+<%
+    /**
+     * Render the email header
```

Comment:
Yes, we need something in place that consolidates this. Again, I'd like to outscope that for this PR